tests/tests_unit/test_data_classes/test_contextualization.py

```diff
@@ -2,15 +2,14 @@
 
 import pytest
 
-from cognite.client import CogniteClient
 from cognite.client.data_classes import ContextualizationJob
 from tests.utils import set_env_var
 
 
 @pytest.fixture()
-def job():
+def job(cognite_client):
     with set_env_var("COGNITE_API_KEY", "BLA"):
-        return ContextualizationJob(job_id=123, status="Queued", cognite_client=CogniteClient())
```

Comment:
Nitpick: this is not related to functions, but we can keep this in this pr

Reply:
Ah woups. I'll move it into another PR.